SerialPrograms/Source/CommonFramework/Notifications/ProgramNotifications.cpp

```diff
@@ -182,14 +180,14 @@ void send_raw_notification(
     );
 #ifdef PA_SLEEPY
     Integration::SleepyDiscordRunner::send_message_sleepy(

```

Comment:
there should be a
If (GlobalSettings::instance().DISCORD.integration.run_on_start && !GlobalSettings::instance().DISCORD.integration.use_dpp)

otherwise, the first move will nuke the object and dpp won't shend shit